### PR TITLE
Id/sort by/btn

### PR DESCRIPTION
**Pull request by Egorka0105**

### CODE REVIEW

File: my-app/src/pages/BodyCube/BodyUsersList/BodyUser/BodyUser.tsx [hunk 1]
```diff
@@ -8,17 +8,26 @@ interface IProps {
...
 			<p className={clN.bodyUser__name}>{name}</p>
-			<p className={clN.bodyUser__ratingChange}>{changePosition}</p>
-			<img className={clN.bodyUser__edit} src={userEditImg} alt="edit user" width="30" height="30" />
+			{changePosition > 0 ? (
+				<p className={clN.bodyUser__ratingChangeUp}>{`${changePosition} place`}</p>
+			) : changePosition < 0 ? (
+				<p className={clN.bodyUser__ratingChangeDown}>{`${changePosition} place`}</p>
+			) : (
+				<p className={clN.bodyUser__ratingChangeDefault}>No Change</p>
+			)}
```
Comment (ZamkovaTetiana) [inline my-app/src/pages/BodyCube/BodyUsersList/BodyUser/BodyUser.tsx]: You don't need conditions for entire elements. They just have different class names, so it is better to implement this logic with "classnames" package. Something like:

```
<p className=cx({
[clN.bodyUser__ratingChangeDown]: changePosition < 0,
...
})>{ changePosition ? `${changePosition} place` : 'No Change'}</p>
```

File: my-app/src/pages/ModalEditUserScore/ModalEditUserScore.tsx [hunk 2]
```diff
@@ -1,18 +1,48 @@
...
+					className={clN.editUser__saveBtn}
+					type="button"
+					onClick={() => {
+						const userData = {
+							id: user?.id,
+							score,
+						};
+						dispatch(editUserScore(userData));
+						closeModal();
+					}}
```
Comment (ZamkovaTetiana) [inline my-app/src/pages/ModalEditUserScore/ModalEditUserScore.tsx]: It is better to move logic to function and write just `onClick={clickHandler}` in markup. Avoid big logic in markup, better move it to the top of the component (above the `return` statement)

File: my-app/src/pages/ModalEditUserScore/modalEditUserScore.module.scss [hunk 3]
```diff
@@ -5,10 +5,24 @@
 
 .modalWrapper {
   position: fixed;
-  @include displayFlex(row, center, center);
+  top: 0;
+  left: 0;
+  display: none;
+  justify-content: center;
+  align-items: center;
```
Comment (ZamkovaTetiana) [inline my-app/src/pages/ModalEditUserScore/modalEditUserScore.module.scss]: `justify-content` and  `align-items`  don't work with `display: none;`

Comment (Egorka0105) [reply my-app/src/pages/ModalEditUserScore/modalEditUserScore.module.scss]: has fixed with mixin

File: my-app/src/pages/ModalEditUserScore/ModalEditUserScore.tsx (hunk 2 above)
Comment (Egorka0105) [reply my-app/src/pages/ModalEditUserScore/ModalEditUserScore.tsx]: ++

File: my-app/src/pages/BodyCube/BodyUsersList/BodyUser/BodyUser.tsx (hunk 1 above)
Comment (Egorka0105) [reply my-app/src/pages/BodyCube/BodyUsersList/BodyUser/BodyUser.tsx]: ++

**Review by ZamkovaTetiana — APPROVED** [review]
mr-bean-thumbs-up

**Pull request merged**